util/topgen/templates/toplevel_pkg.sv.tpl

```diff
@@ -82,6 +82,13 @@ package top_${top["name"]}_pkg;
 % endfor
     ${lib.Name.from_snake_case("top_" + top["name"] + "_outgoing_alert_" + alert_group + "_id_count").as_camel_case()}
   } ${"outgoing_alert_" + alert_group + "_id_e"};
+
+  // Enumeration of ${alert_group} outgoing alerts AsyncOn configuration
+  parameter logic [NOutgoingAlerts${alert_group.capitalize()}-1:0] AsyncOnOutgoingAlert${alert_group.capitalize()} = {
+  % for alert in list(reversed(top["outgoing_alert"][alert_group])):
```

Comment:
Yes. This fails otherwise, probably because of the `loop.last` statement:
```
  File "/Users/robertschilling/Documents/gitprojects/opentitan/util/topgen/templates/toplevel_pkg.sv.tpl", line 89, in render_body
    1'b${alert['async']}${"" if loop.last else ","}
  File "/Users/robertschilling/Documents/gitprojects/opentitan/.venv/lib/python3.13/site-packages/mako/runtime.py", line 331, in last
    return self.index == len(self) - 1
  File "/Users/robertschilling/Documents/gitprojects/opentitan/.venv/lib/python3.13/site-packages/mako/util.py", line 112, in oneshot
    result = self.fget(obj, *args, **kw)
  File "/Users/robertschilling/Documents/gitprojects/opentitan/.venv/lib/python3.13/site-packages/mako/runtime.py", line 319, in __len__
    return len(self._iterable)
TypeError: object of type 'list_reverseiterator' has no len()
```

Reply:
Ohh, I see. That's not super obvious :-D